app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -30,9 +30,21 @@
       <% end %>
     <% end %>
     <% if history.any? %>
+      <% expand_full_history =
+          local_assigns.include?(:expand_full_history) &&
+          expand_full_history &&
+          history.size > 1
+      %>
+      <% if expand_full_history %>
+        <% rest_of_history = history[1..-1] %>
+        <% history = [history.first] %>
+      <% else %>
+        <% rest_of_history = [] %>
+      <% end %>
+
       <div class="change-notes" id="history">
         <h3>Page history</h3>
-        <ol>
+        <ol id="change-notes-history">
```

Comment:
Isn't `change-notes-rest-of-history` required for `aria-` support, rather than `change-notes-history`?